### ISSUE TITLE
Build Docker image for aarch64

### ISSUE THREAD

**Issue by jwillikers** [tryffel/jellycli#28]
Would it be possible to also build the Docker image for 64-bit Arm?

**Comment by tryffel** [comment]
Yes I think it's possible, but it would require a machine with an arm64 architecture to actually test and debug the build, and I don't have such computer available right now. I'll see what I can do about it, though. The build itself is straightforward and does not necessarily require any changes to the application itself, but only to the provided Dockerfile. Feel free to create a pull request with the changes and we'll see how well it works!

**Comment by jwillikers** [comment]
> Yes I think it's possible, but it would require a machine with an arm64 architecture to actually test and debug the build, and I don't have such computer available right now. I'll see what I can do about it, though. The build itself is straightforward and does not necessarily require any changes to the application itself, but only to the provided Dockerfile. Feel free to create a pull request with the changes and we'll see how well it works!

I've got it running on my ROCKPro64 (64-bit Arm) right now and it's been running great so far. No changes were required to build it for `arm64`. I think the only required changes would be in the build pipeline.
